### ISSUE TITLE
[Accessibility]: Tutorial Icons for batch changes are read as `image` by screen readers

### ISSUE THREAD

**Issue by BolajiOlajide** [sourcegraph/sourcegraph#35272]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34068

### Problem description

When navigating through the `Getting Started` section of the `Batch Changes` page - the tutorial section contains icons that are purely presentational and don't have a `title` or `alt` attribute, as such they're read as `image` by the screen reader.

![CleanShot 2022-05-10 at 23 41 05@2x](https://user-images.githubusercontent.com/25608335/167733456-52343d69-7e56-4a19-916d-2e190be30fec.png)

### Expected behavior

Since the icons are mainly presentational, they shouldn't be read out by the screen reader.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
